### PR TITLE
LPS-147854 Testing ocular do not merge

### PR DESCRIPTION
**Pull request by susanchen3**
https://issues.liferay.com/browse/LPS-147854

### CODE REVIEW

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
didn't account for window size here

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
**TLDR:** The first ocular assert with the `InfoButtonBefore.png` image is able to pass. However, the second image `InfoButtonAfter.png` hasn't been able to pass on CI. 

I've tried lowering the match percentage from 80 to 70 to 60 to 50. But no luck.
I've also tried uploading the result image (it's black and white) it produces locally as the baseline image on CI, but it still fails with 
InfoButtonAfter. 
The red is the indication of the difference between the baseline image and the result image. 

My theory as to why it's failing is because the `InfoButtonAfter.png` image has a blue highlighted background whereas `InfoButtonBefore.png` has a white one. I uploaded `ActiveState.png`, which also has a blue highlighted background and it also failed. :thinking:

**Comment by john-co** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
I just pushed the screenshots from my local. I'm not sure what caused the difference from your local @susanchen3 :man_shrugging:

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
pls

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
note to self: change snap to baseline

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile
